design/sketch.webidl

```diff
@@ -5,6 +5,17 @@ typedef long u32;
 // ****************************************************************************
 
 // Buffer
+
+callback WebGPUMappedMemorySuccessCallback = void (ArrayBuffer);
+callback WebGPUMappedMemoryErrorCallback = void (DOMString);
+interface WebGPUMappedMemory {
+    bool isPending();
+    ArrayBuffer getPointer();
```

Comment:
`ArrayBuffer?` (i.e., nullable)? Also, assuming that `SharedArrayBuffer`s will make their comeback to the platform, may be a `SharedArrayBuffer` should be returned?

Reply:
Done